protos/shared/shared.proto

```diff
@@ -19,10 +19,19 @@ option java_package = "io.btfs.shared";
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 import "google/protobuf/timestamp.proto";
 
+service RuntimeService {
+  rpc CheckRuntime(SignedRuntimeInfoRequest) returns (RuntimeInfoReport);
+}
+
 message RuntimeInfoRequest {
   bytes request_address = 1;
   google.protobuf.Timestamp curent_time = 2;
-  bytes signature = 3;
+}
+
+message SignedRuntimeInfoRequest {
+  RuntimeInfoRequest req = 1;
+  bytes signature = 2;
+  google.protobuf.Timestamp curent_time = 3;
```

Comment:
Not need time here any more.

Reply:
@jin-tron @wxue if everyone agrees, then I will delete.

Reply:
okay. i agree!